src/promql/src/functions/changes.rs

```diff
@@ -19,7 +19,6 @@ use std::sync::Arc;
 
 use common_macro::range_fn;
 use datafusion::arrow::array::{Float64Array, TimestampMillisecondArray};
-use datafusion::arrow::datatypes::TimeUnit;
```

Comment:
Well, clippy didn't work for this before?

Reply:
`TimestampMillisecondArray::new_null(0).data_type().clone()` is used to gernerate `Datatype`, So we don't need case like `DataType::Timestamp(TimeUnit::Millisecond, None)` @waynexia